app/models.py

```diff
@@ -1748,6 +1751,23 @@ def _get_letter_cost(self):
 
         return letter_rate
 
+    def _generate_unsubscribe_link(self, base_url):
```

Comment:
It would be useful to add a docstring/comment here explaining why we have admin base URL for body, and api base URL for header maybe?

I gather it's because in header we have one-click POST link, and in body we need a GET link so the email previews do not auto-unsubscribe the user?